source/configuration_manual/push_notification.rst

```diff
@@ -240,23 +240,20 @@ Configuration
 -------------
 
 Lua push notification handler requires :ref:`mail_lua <plugin-mail-lua>` and
-``push_notification_lua`` plugins to be loaded in addition to the plugins 
+``push_notification_lua`` plugins to be loaded in addition to the plugin
 discussed :ref:`above <push_notification-usage>`.
 
-============================== ======== ============= =================================================================
-Name                           Required Type          Description
-============================== ======== ============= =================================================================
-``push_notification_driver``   **YES**  :ref:`string` To identify this settings block the driver should get the value
-                                                      ``lua``.
+============================== ======== ===============  =================================================================
+Name                           Required Type             Description
+============================== ======== ===============  =================================================================
+``push_notification_driver``   **YES**  :ref:`string`    To identify this settings block the driver should get the value
+                                                         ``lua``.
 
-``push_notification_lua_path`` NO       :ref:`string` The lua file to execute. If no script is specified,
-                                                      :dovecot_plugin:ref:`mail_lua_script` will be used by default.
-
-                                                      This setting is optional as the driver first checks the
-                                                      environment ``push_notification_lua_script_path`` and loads it.
-                                                      This also means that this environment variable takes precedence
-                                                      over this setting.
-============================== ======== ============= =================================================================
+``lua_file``                    NO       :ref:`file`     The lua file to execute. If no path is specified it falls back
+                                                         to the globally defined :dovecot_core:ref:`lua_file`
```

Comment:
It's a bit incorrect/unnecessary to say `If no path is specified it falls back to the globally defined ...`, since there's no specific non-global lua_file setting. It's the same setting always, just in different filters. I think it could just say the same as lua_settings below, i.e.:

```
The lua file to execute. See :dovecot_core:ref:`lua_file`
```

Reply:
Right, i felt a bit wrong writing it already :) Fixed now.